Source/App.js

```diff
@@ -110,9 +113,9 @@
                 var latitude = Cesium.Math.toDegrees(cartographicPosition.latitude);
                 var longitude = Cesium.Math.toDegrees(cartographicPosition.longitude);
                 // Modify description
-                var description = '<table class="cesium-infoBox-defaultTable cesium-infoBox-defaultTable-lighter"><tbody>';
-                description += '<tr><th>' + "Latitude" + '</th><td>' + latitude + '</td></tr>';
-                description += '<tr><th>' + "Longitude" + '</th><td>' + longitude + '</td></tr>';
+                description = '<table class="cesium-infoBox-defaultTable cesium-infoBox-defaultTable-lighter"><tbody>';
+                description += '<tr><th>' + "Latitude" + '</th><td>' + latitude.toFixed(5) + '</td></tr>';
```

Comment:
I commented on the tutorial too, but since Cesium is lon/lat, we should keep Longitude first here